profiles/GaryNLOL

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/46727048/146076465-6e91c80e-d2da-44af-b5ef-79857e49c00a.png",
```

Comment:
I recommend more of an image like this.

![image](https://user-images.githubusercontent.com/51391473/146079157-cd2cd253-9d3f-43e1-8223-98e6d370f48f.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/146079157-cd2cd253-9d3f-43e1-8223-98e6d370f48f.png",
```